VirtoCommerce.MarketingModule.Test/DynamicPromotionTests.cs

```diff
@@ -92,4 +75,4 @@ public IEnumerable<Coupon> FindValidCoupons(ICollection<string> couponCodes, str
             }
         }
     }
-}
+}
```

Comment:
.editorconfig has `insert_final_newline = true` - since your code editor doesn't support .editorconfig you should use different editor.